completions/tipc

```diff
@@ -0,0 +1,307 @@
+# tipc(8) completion                                       -*- shell-script -*-
+
+_tipc_media() {
+    local optind=$1
+
+    if [[ $cword -eq $optind ]]; then
+        COMPREPLY=( $( compgen -W 'media' -- $cur) )
+        return 0
```

Comment:
Could you replace all these `return 0` and `return 1` with plain `return` across the whole file as the return values do not seem to be used for anything?